selftests/spell.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh -e
+pylint -j 1 --errors-only --disable=all --enable=spelling --spelling-dict=en_US --spelling-private-dict-file=spell.ignore *
```

Comment:
nice

Reply:
yep, spell errors we need to fix tend to decrease during the release freeze.